.evergreen/start-orchestration.sh

```diff
@@ -16,27 +16,24 @@ echo From shell `date` > $MONGO_ORCHESTRATION_HOME/server.log
 
 cd "$MONGO_ORCHESTRATION_HOME"
 # Setup or use the existing virtualenv for mongo-orchestration.
-#
-# Many of the Linux distros in Evergreen ship Python 2.6 as the
-# system Python. Core libraries installed by virtualenv (setuptools,
-# pip, wheel) have dropped, or soon will drop, support for Python
-# 2.6. Starting with version 14, virtualenv upgrades these libraries
-# to the latest available on pypi when creating the virtual environment
-# unless you pass --never-download.
+# Prefer using Python 3 from the toolchain over the default system python.
+PYTHON=$(command -v /opt/mongodbtoolchain/v2/bin/python3 || command -v python3 || command -v python)
+$PYTHON -c 'import sys; print(sys.version)'
+$PYTHON -c 'import ssl; print(ssl.OPENSSL_VERSION)'
+
 if [ -f venv/bin/activate ]; then
   . venv/bin/activate
 elif [ -f venv/Scripts/activate ]; then
   . venv/Scripts/activate
-elif python -m virtualenv --system-site-packages --never-download venv || virtualenv --system-site-packages --never-download venv; then
```

Comment:
I think we should use maintained Python 3 versions where ever possible and avoid using `--never-download`. @behackett do you see a problem with this new approach?

Reply:
You're assuming the python3 in the server toolchain is still supported by the latest setuptools and pip, or whatever "python3" points to on the OS, or whatever "python" points to on the OS. I don't think you can make that assumption.

Reply:
Okay I added back `--never-download`. It's worth mentioning that `--never-download` defaults to True in recent virtualenv versions. I did not track down the specific version it was changed. But regardless I agree that it will help.